src/config.py

```diff
@@ -1,2 +1,2 @@
-KOLIBRI_PORT = 5000
+KOLIBRI_PORT = 5421
```

Comment:
We don't want to default to 0?

Reply:
No, because doing that properly would involve getting the discovered port from Kolibri. It's possible, but would duplicate work I have in progress and would be a bigger lift.